rocAL_pybind/examples/rocAL_training_example/README.md

```diff
@@ -0,0 +1,113 @@
+# ImageNet training in PyTorch
+
+This implements training of popular model architectures, such as ResNet, AlexNet, and VGG on the ImageNet dataset.
+This version has been modified to use rocAL. It assumes that the dataset is raw JPEGs from the ImageNet dataset. If offers CPU and GPU based pipeline for rocAL - use rocal-cpu switch to enable CPU one and use rocal-gpu switch to enable GPU one.
+
+To run use the following command
+```bash
```

Comment:
Mention this "to run distributed pytorch training"

Reply:
The shared command is for running a training in single node, multi GPU setup. The distributed training commands are shared below in a separate Multiple nodes section